lib/search_config.rb

```diff
@@ -1,5 +1,6 @@
 class SearchConfig
   %w[
+    base_uri
```

Comment:
This looks like a duplicate - it's already in the array on line 10.

Reply:
Removed